settings.gradle

```diff
@@ -192,3 +194,27 @@ if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
   }
 }
 
+dependencyResolutionManagement {
+  versionCatalogs {
+    libs {
+      from(files("./libs.versions.toml"))
+
+      library("delta-standalone", "io.delta:delta-standalone_${scalaVersion}:0.6.0")
+      library("flink115-table-planner", "org.apache.flink:flink-table-planner_${scalaVersion}:1.15.0")
```

Comment:
The Flink and Jackson declarations can be avoided when using the BOM approach:
* Declare a `platform()` dependency to the BOM
* Declare the Flink/Jackson dependency, but w/o the version (because that is declared in the BOM)

Example:
```kotlin
    implementation(platform(libs.flink115.bom))
    implementation("org.apache.flink:flink-table-planner_${scalaVersion}")
```

Reply:
I did it for Jackson but can't find a bom for Flink.